modules/manage/pages/kubernetes/upgrade-kubernetes.adoc

```diff
@@ -9,6 +9,15 @@ Upgrading the Kubernetes version in a cluster ensures that your infrastructure i
 * A running Redpanda deployment on a Kubernetes cluster.
 * Familiarity with your hosting platform (GKE, EKS, AKS, or self-hosted) and any CLI tools your platform provides, such as `gcloud` or `eksctl`.
 * Confirmation that the new version of Kubernetes is compatible with the version of Redpanda you are using. See the xref:reference:redpanda-helm-spec.adoc#requirements[Helm chart requirements].
+* <<Upgrade the Redpanda Helm chart>>.
+
+=== Upgrade the Redpanda Helm chart
+
+Before upgrading Kubernetes, upgrade your Redpanda Helm chart to the latest version. Upgrading the chart ensures that older versions of Kubernetes resources are replaced by newer versions. Failing to do so might lead to Helm encountering deprecated versions during the Kubernetes upgrade, resulting in errors and potential downtime.
+
+For example, starting from Kubernetes 1.25+, the `v1beta1` version of the PodDisruptionBudget resource was deprecated. If you perform a Helm upgrade after updating Kubernetes, Helm carries out a 3-way merge between the desired state, the previous state, and the current state of your cluster. If your cluster's previous state still references the `v1beta1` resource version, Helm will fail to upgrade as this version isn't available in Kubernetes 1.25+.
```

Comment:
```suggestion
For example, starting in Kubernetes 1.25 and later, the `v1beta1` version of the PodDisruptionBudget resource is deprecated. If you were running a previous version of Kubernetes and you performed a Helm upgrade after updating Kubernetes, Helm would carry out a 3-way merge between the desired state, the previous state, and the current state of your cluster. If your cluster's previous state still referenced the `v1beta1` resource version, Helm would fail to upgrade as this version isn't available in your version of Kubernetes.
```